admin_manual/appliance/Clamav.rst

```diff
@@ -78,6 +78,11 @@ If you want to refresh the ClamAV database manually, follow these steps:
   # Start the refresh process again with this command:
   sudo freshclam
 
+..note::
+
+	- When the app is just enabled (without configuration or with wrong configuration) it will strictly reject **all** uploads for the whole instance.
+	- Configuration can be done via occ `config:app:set files_antivirus whatever_key value` to circumvent this issue.
```

Comment:
Proposal:
``Configuration can be done via occ `config:app:set files_antivirus whatever_key value` to circumvent this issue.`` -->
``To Circumvent this issue, you can configure the app via the occ command  set: `config:app:set files_antivirus`. Please use the correct keys and values to work in your environment. For more details how to use the command run: `occ config:app:set --help` ``